README.md

```diff
@@ -182,6 +182,8 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
+Write, compile and run your code. Try it out!
```

Comment:
```suggestion
Write, compile and run your code within a browser. Try it out!
```
@davorpa @Suman373 What do we think of this.

Reply:
Looks better!!